### DISCUSSION TITLE
Can't login to Kibana?

### DISCUSSION THREAD

**Discussion by JaredKeyes** [Security-Onion-Solutions/securityonion#5310] [Unsupported Versions] [answered]
Exact version: **2.3.70**
Install source: **Installed from Security Onion ISO on CentOS 7**
Install type: **Eval**
Does so-status show all services running? **Yes**
Do you get any failures when you run salt-call state.highstate from the node? **603 Succeeded 0 Failed**
Explain your issue: 
I'm trying to access Kibana, but when I click on the tab it prompts for a login and doesn't except my password. I found some troubleshooting steps on the forum which I ran through and I'll list them below:

Execute so-elasticsearch-query / on the manager - Does it succeed? - **It does.**

Review logs in /opt/so/log/kibana/
Review logs in /opt/so/log/elasticsearch/ - **I looked through the logs located in both places and I wasn't sure what I was looking for, but I didn't see anything that stood out to me as an issue.**

Disable Elastic auth via so-elastic-auth false. Can you access Kibana again as an anonymous user? - **Yes, I can gain access as an anonymous user.**
Re-enable Elastic auth via so-elastic-auth true. Does is still list your user as needing to change the password? - **After re-enabling auth, the issue reoccurs.**
Review users listed in /opt/so/conf/elasticsearch/users file. Does it have the correct email addresses listed, with a hash beginning with $2a? - **Yes, my users are listed there and the hash begins with $2a**

The author of the original post said he solved his issue by running soup. I was able to do that, but it didn't solve my issue. Everything else seems to be working fine (This is my first time using it so I'm still learning). Does anyone else have any troubleshooting tips?

Thanks in advance!

**Comment by jertel** [comment, marked as answer]
Your troubleshooting steps look spot on, so it is surprising that you're continuing to have this problem. I think a good next step is to add another user via `so-user-add` and see if it experiences the same problem.

**Reply by JaredKeyes** [reply]
That actually reminded me of some more troubleshooting I've done. This is actually done on a second account. My first account also couldn't get in to Kibana, and it couldn't login to the Hive, and received a 500 error when trying to access data and hunt. The second account can do everything but login to Kibana (everything that I've attempted to do at least), and now with the third account it has access to everything! So, technically, problem solved! Very strange though. Thank you for your help!

Edit: For some reason, even after trying multiple browsers and rebooting my computer, it won't let me mark your answer as THE answer, so maybe security onion just hates me. Either way, thank you for your help!

**Reply by jertel** [reply]
Good to hear you have a working user now. It could be possible to end up in a scenario where a user works in SOC but not TheHive, or other apps, if the system hadn't fully started all processes when the `so-user-add` script was invoke. I can't be certain that's what going on with your system but something to consider.
